.travis.yml

```diff
@@ -141,6 +144,7 @@ matrix:
             - python
             - python2
             - tesseract
+            - libarchive
```

Comment:
When you're adding a new item to an existing list, look to see if the list is in sorted order (as this one was before your addition) - if so, you should probably add the item to preserve that sorted order.

Reply:
Thanks for pointing out, I have changed this.